server_admin/topics/identity-broker/configuration.adoc

```diff
@@ -1,87 +1,69 @@
++
 .IDP login page
 image:{project_images}/identity-provider-login-page.png[]
 
-
 Social::
-  Social providers allow you to enable social authentication in your realm.
-  {project_name} makes it easy to let users log in to your application using an existing account with a social network.
-  Currently supported providers include: Twitter, Facebook, Google, LinkedIn, Instagram, Microsoft, PayPal, Openshift v3, GitHub, GitLab, Bitbucket, and Stack Overflow.
+  Social providers enable social authentication in your realm. With {project_name}, users can log in to your application using a social network account. Supported providers include Twitter, Facebook, Google, LinkedIn, Instagram, Microsoft, PayPal, Openshift v3, GitHub, GitLab, Bitbucket, and Stack Overflow.
 
 Protocol-based::
-  Protocol-based providers are those that rely on a specific protocol in order to authenticate and authorize users.
-  They allow you to connect to any identity provider compliant with a specific protocol.
-  {project_name} provides support for SAML v2.0 and OpenID Connect v1.0 protocols.
-  It makes it easy to configure and broker any identity provider based on these open standards.
+  Protocol-based providers rely on specific protocols to authenticate and authorize users. Using these providers, you can connect to any identity provider compliant with a specific protocol. {project_name} provides support for SAML v2.0 and OpenID Connect v1.0 protocols. You can configure and broker any identity provider based on these open standards.
 
-Although each type of identity provider has its own configuration options, all of them share some very common configuration.
-Regardless of which identity provider you are creating, you'll see the following configuration options available:
+Although each type of identity provider has its configuration options, all share a common configuration. The following configuration options available:
 
 .Common Configuration
 [cols="1,1", options="header"]
 |===
 |Configuration|Description
 
 |Alias
-|The alias is a unique identifier for an identity provider. It is used to reference an identity provider internally.
- Some protocols such as OpenID Connect require a redirect URI or callback url in order to communicate with an identity provider.
- In this case, the alias is used to build the redirect URI.
- Every single identity provider must have an alias. Examples are `facebook`, `google`, `idp.acme.com`, etc.
+|The alias is a unique identifier for an identity provider and references an internal identity provider. {project_name} uses the alias to build redirect URIs for OpenID Connect protocols, which require a redirect URI or callback URL to communicate with an identity provider. All identity providers must have an alias. Alias examples include `facebook`, `google`, and `idp.acme.com`.
```

Comment:
replace ", which" with "that"
because the subordinate cause defines the introductory clause.